core/modules/widgets/navigator.js

```diff
@@ -209,6 +209,10 @@ NavigatorWidget.prototype.handleCloseOtherTiddlersEvent = function(event) {
 
 // Place a tiddler in edit mode
 NavigatorWidget.prototype.handleEditTiddlerEvent = function(event) {
+	var editTiddler = $tw.hooks.invokeHook("th-editing-tiddler",event);
```

Comment:
Here you are using the globally available tiddlywiki. Doesn't the widgets receive a reference to the scoped one on their constructor? If so, better use that one